clientv3/README.md

```diff
@@ -72,6 +72,14 @@ if err != nil {
 }
 ```
 
+## Metrics
+
+etcd client provides metrics reporting. This can enable the operators to know how clients are performing and which client generates the most load.
```

Comment:
@heyitsanthony We can keep this. Just the same opinion from you, can we keep this part as well?
`The etcd client optionally exposes RPC metrics through go-grpc-prometheus. This can enable the operators to know how clients are performing and which client generates the most load. see the examples.`
The only reason i wanted is to make the new users understand it. If you feel otherwise will do the changes as per your suggestion.
Thank you!